httpclient5/src/main/java/org/apache/hc/client5/http/ssl/AbstractClientTlsStrategy.java

```diff
@@ -271,7 +272,7 @@ void verifySession(
                     final X509Certificate x509 = (X509Certificate) cert;
                     final X500Principal peer = x509.getSubjectX500Principal();
 
-                    LOG.debug(" peer principal: {}", peer);
+                    LOG.debug("Escaped peer principal: {}", toEscapedString(peer));
```

Comment:
@arturobernalg "Escaped peer principal" sounds confusing to me. I would just leave it as "Peer principal".